.github/workflows/pr_test.yml

```diff
@@ -127,6 +127,9 @@ jobs:
 
     - name: Provision test environment
       run: |
+        if [[ "${{ matrix.platforms.image }}" == "litmusimage/debian:11" ]]; then
```

Comment:
This change would not be considered a valid fix, since it would result in any local runs continuing to fail.